lint-rules-android-lint/src/test/java/com/vanniktech/lintrules/android/AnnotationOrderDetectorTest.kt

```diff
@@ -613,4 +637,22 @@ class AnnotationOrderDetectorTest {
         .run()
         .expectClean()
   }
+
+  @Test fun kotlinCustomAnnotationOnNullableReturn() {
+    lint()
+        .allowCompilationErrors()
+        .files(kt("""
+            package foo
+
+            annotation class Custom
```

Comment:
Initially I had this in Kotlin but didn't manage to make it fail. It was because of this line missing. In Java sources adding just `Custom` is valid. But in Kotlin, it is just ignored even when `allowCompilationErrors` is set.